editor/src/components/common/notices.tsx

```diff
@@ -27,6 +27,18 @@ export const getStylesForLevel = (level: NoticeLevel): React.CSSProperties => {
   return resultingStyle
 }
 
+export const getPrefixForLevel = (level: NoticeLevel): string => {
+  let resultingPrefix = ''
+  if (level === 'WARNING') {
+    resultingPrefix = '﹗'
+  } else if (level === 'ERROR') {
+    resultingPrefix = '⚠️'
+  } else if (level === 'SUCCESS') {
+    resultingPrefix = '✓'
+  }
+  return resultingPrefix
+}
+
```

Comment:
a switch/case with returns and an `assertNever` in the default case would be more ideal (since it automatically breaks when the `NoticeLevel` type changes), but this is more of a nice-to-have